packages/dd-trace/test/appsec/sdk/user_blocking.spec.js

```diff
@@ -21,8 +29,8 @@ describe('user_blocking', () => {
 
     before(() => {
       const runStub = sinon.stub(waf, 'run')
-      runStub.withArgs({ persistent: { [USER_ID]: 'user' } }).returns(['block'])
-      runStub.withArgs({ persistent: { [USER_ID]: 'gooduser' } }).returns([''])
+      runStub.withArgs({ persistent: { [USER_ID]: 'user' } }).returns(resultActions)
+      runStub.withArgs({ persistent: { [USER_ID]: 'gooduser' } }).returns(undefined)
```

Comment:
wait when it doesn't match any action, should it return with empty object or undefined ?

Reply:
You are right. Changed to an empty object to mimic `waf.run` behaviour.